src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -576,7 +576,16 @@ const generateTaskRole = (frameworkName, taskRole, labels, config, storageConfig
             fieldPath: `metadata.annotations['hivedscheduler.microsoft.com/pod-gpu-isolation']`,
           },
         },
-      });
+      },
+      {
+        name: 'PAI_AMD_VISIBLE_DEVICES',
```

Comment:
Where is the code to consume PAI_AMD_VISIBLE_DEVICES? I.e. your AMD runtime code

Reply:
pls check [here](https://github.com/abuccts/rocm-container-runtime/blob/master/rocm-container-runtime), since drivers and docker are out of the scope of pai, it's not suitable to maintain such a runtime which may have driver/docker issue

Reply:
OK we can discuss this in scrum.
BTW you also need to update PAI_AMD_VISIBLE_DEVICES in your repo

Reply:
@abuccts, how do you deploy the AMD runtime to the bed?

Reply:
> BTW you also need to update PAI_AMD_VISIBLE_DEVICES in your repo

I have made it configurable, the runtime needs a config file like nvidia-container-runtime

> how do you deploy the AMD runtime to the bed?

just copy the binary and config to `/usr/bin` and `/etc`, we could use ansible playbook to copy those two files like installing nvidia runtime